--- ./scripts/airodump-ng-oui-update.oui_destdir~	2009-08-07 22:53:15.000000000 +0200
+++ ./scripts/airodump-ng-oui-update	2011-06-16 18:50:33.000000000 +0200
@@ -4,7 +4,7 @@ CURL=`which curl 2>/dev/null`
 WGET=`which wget 2>/dev/null`
 OUI_DOWNLOAD_URL="http://standards.ieee.org/regauth/oui/oui.txt"
 
-OUI_PATH="/usr/local/etc/aircrack-ng"
+OUI_PATH="/usr/share/aircrack-ng"
 AIRODUMP_NG_OUI="${OUI_PATH}/airodump-ng-oui.txt"
 OUI_IEEE="${OUI_PATH}/oui.txt"
 USERID=""
--- ./src/Makefile.oui_destdir~	2011-06-16 17:40:38.000000000 +0200
+++ ./src/Makefile	2011-06-16 17:40:38.000000000 +0200
@@ -4,6 +4,7 @@ include $(AC_ROOT)/common.mak
 
 TEST_DIR	= $(AC_ROOT)/test
 
+CPPFLAGS	+= -DOUI_PATH='"$(datadir)/aircrack-ng/airodump-ng-oui.txt"'
 CFLAGS		+= -Iinclude
 
 iCC             = $(shell find /opt/intel/cc/*/bin/icc)
--- ./src/airodump-ng.h.oui_destdir~	2010-01-12 01:26:31.000000000 +0100
+++ ./src/airodump-ng.h	2011-06-16 17:40:38.000000000 +0200
@@ -111,7 +111,6 @@ static uchar ZERO[32] =
 "\x00\x00\x00\x00\x00\x00\x00\x00"
 "\x00\x00\x00\x00\x00\x00\x00\x00";
 
-#define OUI_PATH "/usr/local/etc/aircrack-ng/airodump-ng-oui.txt"
 #define MIN_RAM_SIZE_LOAD_OUI_RAM 32768
 
 int read_pkts=0;
