--- aircrack-ng-1.1/scripts/airodump-ng-oui-update.destdir~	2010-08-05 02:34:21.001168811 +0200
+++ aircrack-ng-1.1/scripts/airodump-ng-oui-update	2010-08-05 02:59:37.570447670 +0200
@@ -4,26 +4,28 @@ CURL=`which curl 2>/dev/null`
 WGET=`which wget 2>/dev/null`
 OUI_DOWNLOAD_URL="http://standards.ieee.org/regauth/oui/oui.txt"
 
-OUI_PATH="/usr/local/etc/aircrack-ng"
+OUI_PATH="${DESTDIR}/usr/share/aircrack-ng"
 AIRODUMP_NG_OUI="${OUI_PATH}/airodump-ng-oui.txt"
 OUI_IEEE="${OUI_PATH}/oui.txt"
 USERID=""
 
 
-# Make sure the user is root
-if [ x"`which id 2> /dev/null`" != "x" ]
-then
+if [ -z "${DESTDIR}" ]; then
+    # Make sure the user is root
+    if [ x"`which id 2> /dev/null`" != "x" ]
+    then
 	USERID="`id -u 2> /dev/null`"
-fi
+    fi
 
-if [ x$USERID = "x" -a x$UID != "x" ]
-then
+    if [ x$USERID = "x" -a x$UID != "x" ]
+    then
 	USERID=$UID
-fi
+    fi
 
-if [ x$USERID != "x" -a x$USERID != "x0" ]
-then
+    if [ x$USERID != "x" -a x$USERID != "x0" ]
+    then
 	echo Run it as root ; exit ;
+    fi
 fi
 
 
--- aircrack-ng-1.1/src/airodump-ng.h.destdir~	2010-01-12 01:26:31.000000000 +0100
+++ aircrack-ng-1.1/src/airodump-ng.h	2010-08-05 02:46:50.767447489 +0200
@@ -111,7 +111,6 @@ static uchar ZERO[32] =
 "\x00\x00\x00\x00\x00\x00\x00\x00"
 "\x00\x00\x00\x00\x00\x00\x00\x00";
 
-#define OUI_PATH "/usr/local/etc/aircrack-ng/airodump-ng-oui.txt"
 #define MIN_RAM_SIZE_LOAD_OUI_RAM 32768
 
 int read_pkts=0;
--- aircrack-ng-1.1/src/Makefile.destdir~	2010-08-05 03:15:02.505559976 +0200
+++ aircrack-ng-1.1/src/Makefile	2010-08-05 03:18:07.395670630 +0200
@@ -4,6 +4,7 @@ include $(AC_ROOT)/common.mak
 
 TEST_DIR	= $(AC_ROOT)/test
 
+CPPFLAGS	+= -DOUI_PATH='"$(datadir)/aircrack-ng/airodump-ng-oui.txt"'
 CFLAGS		+= -Iinclude
 
 iCC             = $(shell find /opt/intel/cc/*/bin/icc)
