--- ./common.mak.make_makeup~	2010-02-13 23:32:28.000000000 +0100
+++ ./common.mak	2011-06-16 17:33:15.000000000 +0200
@@ -31,16 +31,16 @@ COMMON_CFLAGS   += -DCYGWIN
 endif
 
 ifeq ($(SQLITE), true)
-    COMMON_CFLAGS	+= -I/usr/local/include -DHAVE_SQLITE
+    COMMON_CFLAGS	+= $(shell pkg-config --cflags sqlite3) -DHAVE_SQLITE -DHAVE_REGEXP
 else
     ifeq ($(sqlite), true)
-        COMMON_CFLAGS	+= -I/usr/local/include -DHAVE_SQLITE
+        COMMON_CFLAGS	+= $(shell pkg-config --cflags sqlite3) -DHAVE_SQLITE -DHAVE_REGEXP
     else
         ifeq ($(SQLITE), TRUE)
-            COMMON_CFLAGS	+= -I/usr/local/include -DHAVE_SQLITE
+            COMMON_CFLAGS	+= $(shell pkg-config --cflags sqlite3) -DHAVE_SQLITE -DHAVE_REGEXP
         else
             ifeq ($(sqlite), TRUE)
-                COMMON_CFLAGS	+= -I/usr/local/include -DHAVE_SQLITE
+                COMMON_CFLAGS	+= $(shell pkg-config --cflags sqlite3) -DHAVE_SQLITE -DHAVE_REGEXP
             endif
         endif
     endif
@@ -66,15 +66,15 @@ AR		= $(TOOL_PREFIX)ar
 REVISION	= $(shell $(AC_ROOT)/evalrev)
 REVFLAGS	= -D_REVISION=$(REVISION)
 
-OPTFLAGS        = -D_FILE_OFFSET_BITS=64
+OPTFLAGS        = $(shell getconf  LFS_CFLAGS)
 CFLAGS          ?= -g -W -Wall -Werror -O3
 CFLAGS          += $(OPTFLAGS) $(REVFLAGS) $(COMMON_CFLAGS)
 
 prefix          = /usr/local
 bindir          = $(prefix)/bin
 sbindir         = $(prefix)/sbin
-mandir          = $(prefix)/man/man1
+mandir          = $(prefix)/man
 datadir         = $(prefix)/share
 docdir          = $(datadir)/doc/aircrack-ng
 libdir		= $(prefix)/lib
-etcdir		= $(prefix)/etc/aircrack-ng 
\ No newline at end of file
+etcdir		= $(prefix)/etc/aircrack-ng 
--- ./manpages/Makefile.make_makeup~	2009-02-20 18:43:30.000000000 +0100
+++ ./manpages/Makefile	2011-06-16 17:33:15.000000000 +0200
@@ -47,13 +47,13 @@ all:
 	@echo Nothing to do
 	
 install: uninstall
-	install -d $(DESTDIR)$(mandir)
-	install -m 644 $(MP_FILES) $(DESTDIR)$(mandir)
+	install -d $(DESTDIR)$(mandir)/man1
+	install -m 644 $(MP_FILES) $(DESTDIR)$(mandir)/man1
 
 uninstall:
-	@-$(foreach CUR_MP,$(MP_FILES), rm -f $(DESTDIR)$(mandir)/$(CUR_MP); echo rm -f $(DESTDIR)$(mandir)/$(CUR_MP); ) 
-	-rm -f $(DESTDIR)$(mandir)/airolib-ng.1
-	-rm -f $(DESTDIR)$(mandir)/wesside-ng.1
-	-rm -f $(DESTDIR)$(mandir)/tkiptun-ng.1
-	-rm -f $(DESTDIR)$(mandir)/buddy-ng.1
-	-rm -f $(DESTDIR)$(mandir)/easside-ng.1
+	@-$(foreach CUR_MP,$(MP_FILES), rm -f $(DESTDIR)$(mandir)/man1/$(CUR_MP); echo rm -f $(DESTDIR)$(mandir)/man1/$(CUR_MP); ) 
+	-rm -f $(DESTDIR)$(mandir)/man1/airolib-ng.1
+	-rm -f $(DESTDIR)$(mandir)/man1/wesside-ng.1
+	-rm -f $(DESTDIR)$(mandir)/man1/tkiptun-ng.1
+	-rm -f $(DESTDIR)$(mandir)/man1/buddy-ng.1
+	-rm -f $(DESTDIR)$(mandir)/man1/easside-ng.1
--- ./scripts/airgraph-ng/common.mak.make_makeup~	2009-02-10 06:05:30.000000000 +0100
+++ ./scripts/airgraph-ng/common.mak	2011-06-16 17:33:15.000000000 +0200
@@ -1,6 +1,6 @@
 prefix          = /usr/local
 bindir          = $(prefix)/bin
-mandir          = $(prefix)/man/man1
+mandir          = $(prefix)/man
 datadir         = $(prefix)/share
 docdir          = $(datadir)/doc/airgraph-ng
 libdir          = $(prefix)/lib
--- ./src/Makefile.make_makeup~	2010-02-13 23:32:28.000000000 +0100
+++ ./src/Makefile	2011-06-16 17:38:09.000000000 +0200
@@ -90,25 +90,25 @@ OBJS_TT		= tkiptun-ng.o common.o crypto.
 
 
 OSD		= osdep
-LIBS		= -L$(OSD) -l$(OSD) $(LIBPCAP) $(LDFLAGS)
+LIBS		= -L$(OSD) -l$(OSD) $(LIBPCAP)
 ifeq ($(OSNAME), cygwin)
 LIBS		+= -liphlpapi -lsetupapi -luuid
 endif
 LIBOSD		= $(OSD)/lib$(OSD).a
 
-LIBSSL		= -lssl -lcrypto $(LDFLAGS)
+LIBSSL		= $(shell pkg-config --libs openssl)
 LIBSQL		=
 ifeq ($(SQLITE), true)
-	LIBSQL		= -L/usr/local/lib -lsqlite3
+	LIBSQL		= $(shell pkg-config --libs sqlite3)
 else
 	ifeq ($(sqlite), true)
-		LIBSQL          = -L/usr/local/lib -lsqlite3
+		LIBSQL          = $(shell pkg-config --libs sqlite3)
 	else
 		ifeq ($(SQLITE), TRUE)
-			LIBSQL          = -L/usr/local/lib -lsqlite3
+			LIBSQL          = $(shell pkg-config --libs sqlite3)
 		else
 			ifeq ($(sqlite), TRUE)
-				LIBSQL          = -L/usr/local/lib -lsqlite3
+				LIBSQL          = $(shell pkg-config --libs sqlite3)
 			endif
 		endif
 	endif
@@ -140,55 +140,55 @@ aircrack-ng-opt-prof_use: $(SRC_AC)
 	aircrack-ng-opt-prof -lpthread $(LIBSQL)
 
 aircrack-ng$(EXE): $(OBJS_AC)
-	$(CC) $(CFLAGS) $(OBJS_AC) $(ASM_AC) -o $(@) -lpthread $(LIBSSL) $(LIBSQL)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_AC) $(ASM_AC) -o $(@) -lpthread $(LIBSSL) $(LIBSQL)
 
 airdecap-ng$(EXE): $(OBJS_AD)
-	$(CC) $(CFLAGS) $(OBJS_AD) -o $(@) $(LIBSSL)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_AD) -o $(@) $(LIBSSL)
 
 packetforge-ng$(EXE): $(OBJS_PF)
-	$(CC) $(CFLAGS) $(OBJS_PF) -o $(@) $(LIBSSL)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_PF) -o $(@) $(LIBSSL)
 
 aireplay-ng$(EXE): $(OBJS_AR) $(LIBOSD)
-	$(CC) $(CFLAGS) $(OBJS_AR) -o $(@) $(LIBS) $(LIBSSL)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_AR) -o $(@) $(LIBS) $(LIBSSL)
 
 airodump-ng$(EXE): $(OBJS_ADU) $(LIBOSD)
-	$(CC) $(CFLAGS) $(OBJS_ADU) -o $(@) $(LIBS) $(LIBSSL) -lpthread
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_ADU) -o $(@) $(LIBS) $(LIBSSL) -lpthread
 
 airserv-ng$(EXE): $(OBJS_AS) $(LIBOSD)
-	$(CC) $(CFLAGS) $(OBJS_AS) -o $(@) $(LIBS) $(LIBPCAP)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_AS) -o $(@) $(LIBS) $(LIBPCAP)
 
 airtun-ng$(EXE): $(OBJS_AT) $(LIBOSD)
-	$(CC) $(CFLAGS) $(OBJS_AT) -o $(@) $(LIBS) $(LIBSSL)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_AT) -o $(@) $(LIBS) $(LIBSSL)
 
 ivstools$(EXE): $(OBJS_IV)
-	$(CC) $(CFLAGS) $(OBJS_IV) -o $(@) $(LIBSSL)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_IV) -o $(@) $(LIBSSL)
 
 kstats$(EXE): kstats.o
-	$(CC) $(CFLAGS) kstats.o -o $(@)
+	$(CC) $(CFLAGS) $(LDFLAGS) kstats.o -o $(@)
 
 wesside-ng$(EXE): $(OBJS_WS) $(LIBOSD)
-	$(CC) $(CFLAGS) $(OBJS_WS) -o $(@) $(LIBS) $(LIBSSL) -lz
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_WS) -o $(@) $(LIBS) $(LIBSSL) -lz
 
 easside-ng$(EXE): $(OBJS_ES) $(LIBOSD)
-	$(CC) $(CFLAGS) $(OBJS_ES) -o $(@) $(LIBS) -lz
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_ES) -o $(@) $(LIBS) -lz
 
 buddy-ng$(EXE): $(OBJS_BUDDY)
-	$(CC) $(CFLAGS) $(OBJS_BUDDY) -o $(@)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_BUDDY) -o $(@)
 
 makeivs-ng$(EXE): $(OBJS_MI)
-	$(CC) $(CFLAGS) $(OBJS_MI) -o $(@)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_MI) -o $(@)
 
 airolib-ng$(EXE): $(OBJS_AL)
-	$(CC) $(CFLAGS) $(OBJS_AL) -o $(@) $(LIBSSL) -DHAVE_REGEXP $(LIBSQL)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_AL) -o $(@) $(LIBSSL) $(LIBSQL)
 
 airbase-ng$(EXE): $(OBJS_AB) $(LIBOSD)
-	$(CC) $(CFLAGS) $(OBJS_AB) -o $(@) $(LIBS) $(LIBSSL) -lpthread
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_AB) -o $(@) $(LIBS) $(LIBSSL) -lpthread
 
 airdecloak-ng$(EXE): $(OBJS_AU)
-	$(CC) $(CFLAGS) $(OBJS_AU) -o $(@)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_AU) -o $(@)
 
 tkiptun-ng$(EXE): $(OBJS_TT) $(LIBOSD)
-	$(CC) $(CFLAGS) $(OBJS_TT) -o $(@) $(LIBS) $(LIBSSL)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_TT) -o $(@) $(LIBS) $(LIBSSL)
 
 strip: $(BINFILES) $(SBINFILES)
 	strip $(BINFILES) $(SBINFILES)
